.github/workflows/ci.yaml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     branches: [develop]
 
+# TODO(Kale-ab) Specify mava container to run tests on.
```

Comment:
TODO still un-done. Is this for a future PR?

Reply:
Yes. Once we have an instadeepct dockerhub account, I will pull a container from there in the workflow. Currently, we are using my personal dockerhub so I haven't added it here.